Improve documentation

Naylin15 opened the pull request:
## Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [Conventional Commit](http://conventionalcommits.org/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue #N/A


## What is the new behavior?
* Improve the following documentation:

  * Contributing guidelines
  * Issue template
  * PR template

* Add YAML file for bot settings

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

Naylin15 commented:
Hi, @DIOHz0r 

I changed the PR template as suggested, it no longer has the checklist for the requirements (commit guidelines), instead is a list, this way we keep the reminder but avoid the checking time

DIOHz0r commented on .github/CONTRIBUTING.md:
```diff
@@ -1,93 +1,146 @@
+
+## Tests
+
+* Make sure test-suite passes: `phpuinit`
```
glpi-plugin doesn't use php-unit, it uses [atoum](http://atoum.org/)

Naylin15 replied:
Ok, changing...

DIOHz0r commented on .github/CONTRIBUTING.md:
````diff
@@ -1,93 +1,150 @@
 
-[commit-message-format]: https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y
+```console
+  npm test
````
For testing the correct commands are:

  - vendor/bin/atoum -ft -bf tests/bootstrap.php -d tests/suite-install -ncc
  - vendor/bin/atoum -ft -bf tests/bootstrap.php -d tests/suite-unit --nccfc CommonTreeDropdown CommonDropdown CommonDBTM CommonGLPI
  - vendor/bin/atoum -ft -bf tests/bootstrap.php -d tests/suite-integration --nccfc CommonTreeDropdown CommonDropdown CommonDBTM CommonGLPI
  - vendor/bin/atoum -ft -bf tests/bootstrap.php -d tests/suite-uninstall --ncc
  - vendor/bin/phpcs -p --standard=vendor/glpi-project/coding-standard/GlpiStandard/ *.php install/ inc/ front/ ajax/ tests/; 

Or you can referrer to "script" section on .travis.yml file for more details.

Naylin15 replied:
Thanks, I'll make the changes

btry replied:
Why not provide some scripts for windows and linux to ease unit tests ?

ajsb85 commented on .github/settings.yml:
```diff
@@ -0,0 +1,46 @@
+  - name: help wanted
+    color: 33aa3f
+  - name: good first issue
+    color: 7057ff
```
@btry No more labels to add?

btry replied:
I suggest we add (some of) the labels I created for this repo. I think most of them may be used for all main repos.

This is suggestion, not mandatory.

Naylin15 replied:
All right, I'll add the missing labels to this PR and check the other settings file

Since you said some of, is there one in particular you want to exclude @btry ?

btry replied:
Nope. As you have a view on documentation of all repos, I think you'll be more efficient to decide which ones should be kept and which ones should be removed.

Naylin15 commented:
I added the missing labels however I removed the WIP one since this will be specified in the contributing guidelines (also in this PR), and changed the repeated colors of some
